delphi-checks/src/main/java/au/com/integradev/delphi/checks/RedundantInheritedCheck.java

```diff
@@ -40,11 +46,78 @@ public class RedundantInheritedCheck extends DelphiCheck {
   @Override
   public DelphiCheckContext visit(RoutineImplementationNode routine, DelphiCheckContext context) {
     for (DelphiNode violationNode : findViolations(routine)) {
-      reportIssue(context, violationNode, MESSAGE);
+      context
+          .newIssue()
+          .onNode(violationNode)
+          .withMessage(MESSAGE)
+          .withQuickFixes(getQuickFixes(violationNode))
+          .report();
     }
     return super.visit(routine, context);
   }
 
+  private static List<QuickFix> getQuickFixes(DelphiNode violationNode) {
```

Comment:
@fourls, wondering what your thoughts are on this quickfix implementation.

Looks like it:
* checks for the previous statement and starts the deletion range from the end of that statement (skipping a trailing semicolons)
* skips a trailing semicolons on the `inherited` statement and ends the deletion range there

This seems mostly fine, but it means that we'll consume comments preceding the `inherited` statement, if they exist.

Reply:
The same argument could be made for compiler directives on a side note.